app/models/post.rb

```diff
@@ -2,4 +2,14 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   has_many :comments
   has_many :likes
+
+  after_create :update_post_counter
+
+  def update_post_counter
+    user.increment!(:posts_counter)
+  end
+
```

Comment:
- [ ] Well done using a callback method, It will be better to make it `private`.
````ruby
 after_create :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
````

Reply:
Solved :+1: